src/builders/libraryBuilder.ts

```diff
@@ -75,7 +75,37 @@ export class LibraryBuilder {
     return resources;
   }
 
-  buildLibrary(): string {
+  createLibraryResource(): R4.ILibrary {
```

Comment:
We should add the list of valuesets defined or used by the IG to the Library resource the way that DaVinci does it as well.   These are contained in the dataRequirement attribute.   This can be a separate Jira task to add them as well.

Reply:
Ah yeah good point. I might want to pawn this off to a new task, because it may involve some restructuring of how we access the ValueSet map and create the library resource.. seeing as each entry in `dataRequirement` needs a resourceType. So we'd probably need to push to the array when we process all the cql definitions that use a ValueSet?

I haven't thought too too much about it, but my gut is saying separate task.